Fix issues reported by asciidoctor-dita-vale (Managing Security Compliance) (#4201)
theforeman/foreman-documentation: guides/common/modules/proc_listing-available-scap-contents-using-web-ui.adoc

:_mod-docs-content-type: PROCEDURE

[id="listing-available-scap-contents-using-web-ui"]

> = Listing available SCAP contents using {ProjectWebUI}

Reviewer: ```suggestion
= Listing available SCAP contents by using {ProjectWebUI}
```
I still feel strongly about this :)

> Use this procedure to view what SCAP contents are already loaded in {Project}.

Reviewer: ```suggestion
You can view what SCAP contents are already loaded in {Project}.
```
